### PR TITLE
Draft: Ambassadors page

### PR DESCRIPTION
**Pull request by ewels**
TODO
- [x] Phil wants to slightly enlarge social media icons at the bottom fo the cards
- [ ] Also to add some classes to the cards content (flex, ...)

### CODE REVIEW

File: content/ambassadors.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        <h4>Recognition</h4>
+        <p>Receive a certificate for your work and appear on the ambassador page of our website.</p>
+    </div>
+    <div class="col-6 col-md-4">
+        <h4>Support</h4>
+        <p>Gain access to an exclusive travel fund to go to events; get stickers and support (e.g., pizza) for events that you organize.</p>
+    </div>
+</div>
+
+<p>Oh, and did we mention that there's exclusive swag?</p>
```
Comment (christopher-hakkaart) [inline content/ambassadors.html:44]:
```suggestion
```

File: content/ambassadors.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    </div>
+</div>
+<div class="row">
+    <div class="col-6 col-md-4">
+        <h4>Involved</h4>
+        <p>Be kept up-to-date regarding our events and have opportunities to be involved in their planning. Be informed of strategic events that you can attend!</p>
+    </div>
+    <div class="col-6 col-md-4">
+        <h4>Recognition</h4>
+        <p>Receive a certificate for your work and appear on the ambassador page of our website.</p>
```
Comment (christopher-hakkaart) [inline content/ambassadors.html:36]:
```suggestion
        <p>Receive a certificate for your work and appear on the ambassador page of our website. Oh, and did we mention that there's exclusive swag?</p>
```

**Review by christopher-hakkaart — CHANGES_REQUESTED** [review]
A line of text is in the wrong place?

**Comment by ewels** [comment]
> A line of text is in the wrong place?

That's how it was in the google doc 🤷🏻‍♂️  But I agree, it's better how you had it. Updated.

**Comment by ewels** [comment]
Hopefully will have some design love before launch, but ready to go now in any case.

**Review by christopher-hakkaart — APPROVED** [review]
🤠

**Comment by ewels** [comment]
⚠️  Updates on google docs, don't merge until applied here.

File: content/ambassadors.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    <li>Share our information reliably and appropriately   </li>
+    <li>Have already participated in some of our events and/or initiatives </li>
+    <li>Are willing to actively support the community as an ambassador for at least one year   </li>
+</ul>
+
+<p>We review our ambassadors annually, so it's no problem to pick up and put down the title as your other commitments allow.</p>
+
+<p>To become an ambassador, please register your interest using the link below. We will get in touch with you soon to chat about the next steps.</p>
+
+<p style="text-align:center; margin: 3rem 0;"><a href="https://seqera.typeform.com/to/CJpqNIks" target="_blank" class="btn btn-color btn-xxl">Register now</a>
```
Comment (ewels) [inline content/ambassadors.html:99]:
```suggestion
<p style="text-align:center; margin: 3rem 0;"><a href="https://seqera.typeform.com/ambassadors" target="_blank" class="btn btn-color btn-xxl">Register now</a>
```

File: content/ambassadors.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+type=page
+status=published
+~~~~~~
+
+<h1>Become an ambassador!</h1>
+
+<p>
+    <img src="/img/nextflow_ambassador_logo.svg" width=400 class="pull-right" style="padding: 2rem 1rem;">
+    The Nextflow community is a dynamic ecosystem that brings together thousands of researchers, developers, and enthusiasts who are passionate about data-driven workflows and scientific computing. It thrives on the principles of open-source software, open science, and knowledge sharing. Here, you'll have the opportunity to connect with like-minded individuals from around the world, exchange ideas, and contribute to the advancement of cutting-edge bioinformatics, genomics, and data analysis pipelines. Whether you're a seasoned expert or just beginning your journey, the Nextflow community offers a supportive environment that values your insights and encourages your growth.
+    <a href="/our_ambassadors.html" target="_blank" class="btn btn-color btn-xxl">Check our ambassadors here!</a>
```
Comment (ewels) [inline content/ambassadors.html:12]:
```suggestion
</p>
<p style="text-align:center;">
    <a href="/our_ambassadors.html" class="btn btn-color btn-xxl">Check our ambassadors here!</a>
```

**Pull request merged**